spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -26,29 +17,41 @@
         end
       end
 
-      context "when saledate is in the 22/23 collection year" do
+      context "when saledate is in the open collection year" do
         let(:record) { build(:sales_log, saledate: Time.zone.local(2023, 1, 1)) }
 
+        before do
+          allow(Time).to receive(:now).and_return(Time.zone.local(2023, 1, 10))
+        end
```

Comment:
I don't think freezing time here (essentially - I think the difference between this and timecop is that timecop may also mock some other stuff?) is helping - if the context here is a general "saledate is in the open collection year" then could that just be the saledate being (real) now?

Reply:
yup